Timing support (#3363)
verilator/verilator: docs/guide/exe_verilator.rst

   If the design is not to be completely Verilated see also the
   :vlopt:`--bbox-sys` and :vlopt:`--bbox-unsup` options.

.. option:: --main

   Generates a simple main C++ file. Without :vlopt:`--timing`, you need to
   modify this file to provide some stimuli to the design. However, this option
   is especially useful with :vlopt:`--timing` and delay-generated clocks, as
   then the main file provides a timing-enabled eval loop and requires no
   modification by the user. :vlopt:`--build` can then be used to build the

>    simulation, allowing you to use Verilator without touching a C++ toolchain.

Reviewer: ```suggestion
   simulation, allowing you to use Verilator without directly invoking
   the C++ toolchain.
```

   mtasks the model is to be partitioned into. If unspecified, Verilator
   approximates a good value.

.. option:: --no-timing

   Disables support for timing constructs such as delays, event controls (unless
   it's at the top of a process), wait statements, and joins.

> 

Reviewer 2: Move down to alphabetize under "timing"
